augmentedreality/Common/3rdparty/arcore/include/arcore_c_api.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Google Inc. All Rights Reserved.
+ * Copyright 2017 Google LLC
  *
```

Comment:
File permissions were changed for this file. Please keep them as they were (100755)

Reply:
This comes from https://github.com/google-ar/arcore-android-sdk/blob/master/libraries/include/arcore_c_api.h. Are you sure we should keep the old version?

Reply:
No, we must keep the new file, but retain the previous file permissions.

![Screen Shot 2022-11-22 at 11 47 23 AM](https://user-images.githubusercontent.com/5695392/203407566-a48c543f-d8a4-404f-986a-df9f8ff3df5e.png)